### DISCUSSION TITLE
Difference beetween TARGET_WAV and SOURCE_WAV parameters.

### DISCUSSION THREAD

**Discussion by Thomcle** [coqui-ai/TTS#3717] [General Q&A]
When looking at the differents optional parameter through `tts -h`, I see those two parameter which differences are not clear:
--target_wav TARGET_WAV
                        Target audio file to convert in the voice of the source_wav
--source_wav SOURCE_WAV
                        Original audio file to convert in the voice of the target_wav
                        
                        
 Can someone explain in details the differences ?

**Comment by eginhard** [comment]
For [voice conversion](https://docs.coqui.ai/en/latest/inference.html#example-voice-conversion-converting-speaker-of-the-source-wav-to-the-speaker-of-the-target-wav), the voice in `source_wav` will be converted to sound like that from `target_wav`.
